sdk/state/open.go

```diff
@@ -2,19 +2,29 @@ package state
 
 import (
 	"fmt"
+	"strconv"
 
 	"github.com/stellar/experimental-payment-channels/sdk/txbuild"
 	"github.com/stellar/go/txnbuild"
 	"github.com/stellar/go/xdr"
 )
 
+// TODO - should store on channel like Update and Close proposals?
```

Comment:
added this TODO for another PR. created a ticket for it: https://github.com/stellar/experimental-payment-channels/issues/92